skyportal/facility_apis/lt.py

```diff
@@ -0,0 +1,325 @@
+import time
+
+from lxml import etree
+from suds import Client
+from dateutil.parser import parse
+from datetime import datetime
+
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+from astropy.io import ascii
+
+from . import FollowUpAPI
+from baselayer.app.access import auth_or_token
+from baselayer.app.env import load_env
+import requests
+from requests_toolbelt.utils import dump
+
+
+LT_SETTINGS = {
+    'USERNAME': '',
+    'PASSWORD': '',
+    'LT_HOST': '',
+    'LT_PORT': '',
+    'MAX_AIRMASS': 2.0,
```

Comment:
Can we make these configurable?

Reply:
@dannygoldstein https://rjsf-team.github.io/react-jsonschema-form/ float sliders don't seem to exist in this plugin... we ok with text boxes?